rotkehlchen/chain/ethereum/decoding/decoder.py

```diff
@@ -522,6 +564,30 @@ def _maybe_enrich_transfers(  # pylint: disable=no-self-use
 
         return None
 
+    def _enrich_protocol_tranfers(  # pylint: disable=no-self-use
+            self,
+            token: EthereumToken,
+            tx_log: EthereumTxReceiptLog,
+            transaction: EthereumTransaction,
+            event: HistoryBaseEntry,
+            action_items: List[ActionItem],
+    ) -> None:
+        """
+        Decode special transfers made by contract execution for example at the moment
+        of depositing assets or withdrawing.
+        It assumes that the event being decoded has been already filtered and is a
+        transfer.
+        """
+        if maybe_enrich_pickle_transfers(
+            token=token, tx_log=tx_log,
+            transaction=transaction,
+            event=event,
+            action_items=action_items,
+        ):
+            return None
```

Comment:
When the only return is `None` just use `return`. It's the same and takes less typing

Reply:
I knew that of the return but thought on explicit better than implicit